### PR TITLE
SEED 757: Start screen - template 

### PR DESCRIPTION
**Pull request by Mitso**

### CODE REVIEW

**Comment by Mitso** [comment]
@peterp @saeedmb8 please review

File: molo/usermetadata/templates/persona.html [hunk 1]
```diff
@@ -2,18 +2,23 @@
 {% load wagtailimages_tags i18n %}
 
 {% block content %}
-<div>
-    <ul>
+<div class="personae">
+    <h3 class="personae__title">Are you a ...</h3>
```
Comment (peterp) [inline molo/usermetadata/templates/persona.html]: this could be an `h1`

File: molo/usermetadata/templates/persona.html [hunk 2]
```diff
@@ -2,18 +2,23 @@
...
-          <li>
-            {% if persona.image %}
-              {% image persona.image width-75 as tmp_photo %}
-              <img src="{{ tmp_photo.url }}" alt="{{ persona.image.title }}"/>
-            {% endif %}
-            <a href="{% url 'molo.usermetadata:set_persona' persona.id %}?next={{ next }}">{{ persona.title }}</a>
+          <li class="personae-list__item">
+              {% if persona.image %}
+                {% image persona.image width-65 as tmp_photo %} 
+                <img src="{{ tmp_photo.url }}" alt="{{ persona.image.title }}" class="personae-icon" />
```
Comment (peterp) [inline molo/usermetadata/templates/persona.html]: could we call this `personae__image`

File: molo/usermetadata/templates/persona.html [hunk 3]
```diff
@@ -2,18 +2,23 @@
...
+              {% endif %}
+            <a href="{% url 'molo.usermetadata:set_persona' persona.id %}?next={{ next }}">
+              <span class="personae-nav">{{ persona.title }}</span>
```
Comment (peterp) [inline molo/usermetadata/templates/persona.html]: please remove the span, and call this `personae__title`

File: molo/usermetadata/templates/persona.html (hunk 1 above)
Comment (peterp) [inline molo/usermetadata/templates/persona.html]: perhaps rename this to `heading heading--personae`, and place it outside `div.personae`

File: molo/usermetadata/templates/persona.html [hunk 4]
```diff
@@ -2,18 +2,23 @@
...
+            </a>
           </li>
         {% endfor %}
     </ul>
-    <a href="{% url 'molo.usermetadata:skip_persona'%}?next={{ next }}">{% trans "Skip" %}</a>
+    <a href="{% url 'molo.usermetadata:skip_persona'%}?next={{ next }}" class="personae-skip">{% trans "Skip" %}</a>
```
Comment (peterp) [inline molo/usermetadata/templates/persona.html]: `personae__skip`

**Comment by Mitso** [comment]
@peterp updated, thanks

File: molo/usermetadata/templates/persona.html [hunk 5]
```diff
@@ -2,18 +2,23 @@
 {% load wagtailimages_tags i18n %}
 
 {% block content %}
-<div>
-    <ul>
+<div class="personae">
+    <h1 class="heading heading--personae">Are you a ...</h1>
```
Comment (peterp) [inline molo/usermetadata/templates/persona.html]: please place this outside of `div.personae`

**Comment by peterp** [comment]
Just one more change, then 👍

**Pull request merged**